src/components/object-definition.ts

```diff
@@ -27,6 +27,7 @@ export function getObjectDefinition(
     realType.flags & ts.TypeFlags.Number ||
     isArrayType(realType) ||
     realTypeName === 'HTMLElement' ||
+    realTypeName === 'Highcharts.Point' ||
```

Comment:
To make your life simpler, I still recommend checking `realTypeName.split('.')[0]` only to avoid editing this check any time someone uses a new type from Highcharts

Reply:
Makes total sense, updated!